best-practices-trove-project-creation.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Creating projects for Trove"
+permalink: /best-practices-trove-project-creation/
+---
+# Creating a data collection project in Trove
+
+With Trove, you can create AI/ML data collection projects through trusted connections between you and the people who contribute to your projects, leading to higher quality data. This document explains how to create such a project in Trove.
+
+## Create Project in Trove Studio
+
+Start your data collection project by signing into the [Trove Studio](https://trove-studio.microsoft.com/) as a requester.  Once your account has been verified, click the "Create project" link to provide the details of your project.
+
+## Provide Clear and Concise Description and Details
+
+**Project Name** - keep this brief but concise ideally containing the object(s) and environment desired for example "Cigarettes on Beaches," "Clothing Stores" or "Pleasant Memories - Food."
+
+**Description** - provide a description in prose form highlighting the key objects in the desired context and environments to be captured in the photos you are looking for in as much detail as possible.  Also call out what the types of photos will _NOT_ be accepted.  Please note our requirement to indicate if the photos you desire will contain face data.  For example - "Send photos of a SPAM can in multiple places in your house (kitchen, living room, bedroom, bathroom). In each place, take at least 5 photos from different sides of the can (front, back, top, left, and right).  Do not send photos of a SPAM can on a shelf in a grocery store or photos containing people." 
```

Comment:
This needs a bit of work. We need to encourage people to write normal people understandable descriptions. I would rewrite this to make that clear. I understand what "prose form" means but it's also too complex for a lot of people that read this quickly to understand.

We don't support faces for 3P so we should say that we don't support it.

Reply:
Great feedback as I struggled with this section for a couple of days to use simple, everyday language to provide guidance to buyers on writing their descriptions.  I agree this needs to be clear and simple so I plan to re-write this section incorporating feedback with the goal of a consistent one voice throughout the documentation.  I didn't add more about the face data because I wasn't sure if the checkbox on face data will show up only for certain types of buyers or all of them.  If we are not supporting faces for 3P, why would we have the checkbox on the self-service project creation page?

Reply:
Added an IMPORTANT note to the section on description that we do not support collection of face data.